### PR TITLE
Require drush 11

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -36,10 +32,10 @@
         }
     ],
     "require": {
-        "thunder/thunder-distribution": "~6.0@STABLE",
+        "thunder/thunder-distribution": "~6.3@STABLE",
         "drupal/core-composer-scaffold": "^9.0",
         "composer/installers": "^1.2",
-        "drush/drush": "^10.3",
+        "drush/drush": "^11.0",
```
Comment (ol0lll) [inline composer.json:38]: Thats not allow, thats require, allow would be
`        "drush/drush": "^10.3||^11.0",`
Do we even need drush as a requirement?

Comment (chrfritsch) [reply composer.json:38]: IMHO, I think it's best practice to work with drush, so we should ship it.

Comment (ol0lll) [reply composer.json:38]: I dont know core/recommended-project does not. You might not need it in at least some of your containers?

Comment (chrfritsch) [reply composer.json:38]: We are referencing to drush in many places all around our documentation. Would be strange to not have it by default

Comment (ol0lll) [reply composer.json:38]: Might be require-dev at max.

Comment (dbosen) [reply composer.json:38]: Drush is used on production

**Pull request closed without merging**